samples/type_ahead_search_adaptive_cards/csharp/README.md

```diff
@@ -0,0 +1,87 @@
+---
+page_type: sample
+description: Demonstrating feature of type ahead search (static and dynamic) control in Adaptive cards.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "11-11-2021 23:30:17"
+---
+# Type ahead search control in Adaptive cards
+
+This sample shows feature of type ahead search (static and dynamic) control in Adaptive cards.
```

Comment:
This sample shows the feature of type ahead search (static and dynamic) control in Adaptive Cards.